connector.yaml

```diff
@@ -2,7 +2,7 @@ version: "1.0"
+  description: "## Source\n\nThe Dropbox Source Connector reads files from a configured Dropbox path and converts \nthem into `opencdc.Record` that can be processed by Conduit. Files larger than \n`fileChunkSizeBytes` (maximum value 4MB) are split into smaller chunks, and each \nchunk is emitted as a separate record.\n\n### Snapshot (Initial Sync)\n\nWhen the connector starts without a saved position, it triggers a snapshot listing \nall files in the configured Dropbox path. A cursor and the last processed timestamp \nare saved at the end of snapshotting.\n\n### Change Data Capture\n\nAfter the snapshot, the connector uses Dropbox's Longpoll API to wait for changes \n(file creation, modification, deletion). Upon detecting changes, it fetches updated \nentries using the saved cursor. If Dropbox reports expired cursor the connector \nfalls back to a fresh scan skipping already-processed files based on the \n`lastProcessedUnixTime` field.\n\nEach record have following metadata fields to support downstream file reassembly:\n\n* `filename`: Name of the file with extension.\n* `file_id`: Unique Dropbox file ID.\n* `file_path`: Full path of the file in Dropbox (e.g., `/Photos/Vacation/image.jpg`).\n* `opencdc.collection`: Path to the parent directory containing the file (e.g., `/Photos/Vacation`).\n* `file_size`: Integer size of the file.\n* `is_chunked`: \"true\" if the file was split into multiple chunks; otherwise false or not present.\n* `chunk_index`: Index of the current chunk (starting from 1) – only present for chunks.\n* `total_chunks`: Total number of chunks – only present for chunked files.\n* `hash`: A hash of the file content. This field can be used to verify data integrity.\n\n## Destination\n\nA destination connector pushes data from upstream resources to an external\nresource via Conduit.\n\nEach record should have following metadata fields:\n\n* `filename`: Name of the file with extension.\n* `opencdc.collection`: Path to the directory containing the file.\n* `file_size`: Integer size of the file.\n* `is_chunked`: \"true\" if the file was split into multiple chunks; otherwise false or not present.\n* `chunk_index`: Index of the current chunk (1-based, required only for chunked files).\n* `total_chunks`: Total number of chunks (required only for chunked files).\n* `hash`: A hash to uniquely identify the file.\n"
```

Comment:
This block appears in this way because there's a trailing space. We will fix that in the next version of the SDK. But for now, you can make this block appear in a nice way by removing any trailing spaces.